src/components/pages/unmarkedWinterStoragePage/UnmarkedWinterStoragePage.tsx

```diff
@@ -110,15 +114,15 @@ const UnmarkedWinterStoragePage = ({
           steps={steps}
         />
       </KoroSection>
-      <Container>
-        <Row>
-          <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
-            <div className="vene-unmarked-winter-storage-page__buttons">
-              <Button className="vene-unmarked-winter-storage-page__button">Jatka</Button>
-            </div>
-          </Col>
-        </Row>
-      </Container>
+      <div className="vene-unmarked-winter-storage-page__buttons">
```

Comment:
the button could take the whole width in mobile (like in the next step):
<img width="314" alt="Screenshot 2020-08-31 at 14 39 31" src="https://user-images.githubusercontent.com/1407958/91716471-688f4e80-eb98-11ea-9f6a-c0f37b46ae40.png">

Reply:
Done 👍📱